processing/src/main/java/io/druid/segment/IndexSpec.java

```diff
@@ -98,7 +99,7 @@ public IndexSpec(
     Preconditions.checkArgument(metricCompression == null || COMPRESSION_NAMES.contains(metricCompression),
                                 "Unknown compression type[%s]", metricCompression);
 
-    this.bitmapSerdeFactory = bitmapSerdeFactory != null ? bitmapSerdeFactory : IndexIO.CONFIGURED_BITMAP_SERDE_FACTORY;
+    this.bitmapSerdeFactory = bitmapSerdeFactory != null ? bitmapSerdeFactory : new ConciseBitmapSerdeFactory();
```

Comment:
The property is in the docs now, and not marked deprecated, so maybe it's worth keeping it but marking it as deprecated now. Or if we are ok with removing the config now, then it should also be removed from the docs.